docs/guides/docs/running-a-node/running-a-beta-5-node.mdx

```diff
@@ -0,0 +1,202 @@
+/>
+
+<TextImport
+  file="../installation/index.mdx"
+  comment="install_testnet"
+  commentType="{/*"
+/>
+
+<CodeImport
+  file="../installation/index.mdx"
+  comment="install_testnet_command"
+  commentType="{/*"
+  lang="sh"
+  trim="true"
+/>
+
+<TextImport
+  file="../installation/index.mdx"
+  comment="set_default_testnet"
+  commentType="{/*"
+/>
+
+<CodeImport
+  file="../installation/index.mdx"
+  comment="set_default_testnet_command"
+  commentType="{/*"
+  lang="sh"
+  trim="true"
+/>
+
+<TextImport
+  file="../quickstart/building-a-smart-contract.mdx"
+  comment="install_help"
+  commentType="{/*"
+/>
+
+## Getting a Sepolia (Ethereum Testnet) API Key
+
+<TextImport
+  file="./running-a-beta-4-node.mdx"
+  comment="get_ethereum_api_key"
+  commentType="{/*"
+/>
+
+### Infura
+
+<CodeImport
+  file="./running-a-beta-4-node.mdx"
+  comment="infura_key"
+  commentType="//"
+  lang="sh"
+  trim="true"
+/>
+
+### Alchemy
+
+<CodeImport
+  file="./running-a-beta-4-node.mdx"
+  comment="alchemy_key"
+  commentType="//"
+  lang="sh"
+  trim="true"
+/>
+
+<TextImport
+  file="./running-a-beta-4-node.mdx"
+  comment="get_ethereum_api_key_end"
+  commentType="{/*"
+/>
+
+## Generating a P2P Key
+
+<TextImport
+  file="./running-a-beta-4-node.mdx"
+  comment="generate_fuel_p2p_key"
+  commentType="{/*"
+/>
+
+{/*TODO: Double check this is still valid*/}
+
+<CodeImport
+  file="./running-a-beta-4-node.mdx"
+  comment="generate_fuel_p2p_key_command"
+  commentType="//"
+  lang="sh"
+  trim="true"
+/>
+
+<TextImport
+  file="./running-a-beta-4-node.mdx"
+  comment="generate_fuel_p2p_key_end"
+  commentType="{/*"
+/>
+
+## Chain Configuration
+
+{/*TODO: Update link for beta-5*/}
+
+To run a local node with persistence, you must configure a `chainConfig.json` file. Here is the [specific configuration](https://github.com/FuelLabs/fuel-core/blob/v0.20.4/deployment/scripts/chainspec/beta_chainspec.json) for `beta-5`:
+
+<CodeImport
+  file="../../../fuel-core/deployment/scripts/chainspec/beta_chainspec.json"
+  lang="json"
+/>
+
+## Running a Local Node
+
+<TextImport
+  file="./running-a-beta-4-node.mdx"
+  comment="running_a_local_node"
+  commentType="{/*"
+/>
+
+{/*TODO: Update contracts and links for beta-5*/}
+
+```sh
+fuel-core run \
+--service-name {ANY_SERVICE_NAME} \
+--keypair {P2P_SECRET} \
+--relayer {ETH_RPC_ENDPOINT}\
+--ip 0.0.0.0 --port 4000 --peering-port 30333 \
+--db-path  ~/.fuel_beta5 \
+--chain ./chainConfig.json \
+--utxo-validation --poa-instant false --enable-p2p \
+--min-gas-price 1 --max-block-size 18874368  --max-transmit-size 18874368 \
+--bootstrap-nodes /dns4/p2p-beta-4.fuel.network/tcp/30333/p2p/16Uiu2HAm3xjsqASZ68KpaJPkPCMUiMgquhjyDHtxcVxVdFkMgRFf,/dns4/p2p-beta-4.fuel.network/tcp/30334/p2p/16Uiu2HAmJyoJ2HrtPRdBALMT8fs5Q25xVj57gZj5s6G6dzbHypoS \
+--sync-header-batch-size 1000 \
+--enable-relayer \
+--relayer-v2-listening-contracts 0xE6B0E27F85abaCfC5149642c30F4BE9a878Aa4e9 \
+--relayer-da-deploy-height 8538323 \
```

Comment:
The same here, we need actual values @tirkesi

Reply:
```
      relayerV2ListeningContracts:
        - 0x557c5cE22F877d975C2cB13D0a961a182d740fD5 # FuelMessagePortal
      relayerDADeployHeight: 4867877
```